package.json

```
@@ -22,11 +22,13 @@
   "scripts": {
     "test": "jest --no-color",
     "lint": "eslint .",
+    "typecheck": "tsc --noEmit",
```

```suggestion
    "typecheck": "tsc",
```

`emitDeclarationOnly` in the config file should be enough